Incremental.c

```diff
@@ -1700,16 +1701,25 @@ int IncrementalRemove(char *devname, char *id_path, int verbose)
 		pr_err("incremental removal requires a kernel device name, not a file: %s\n", devname);
 		return 1;
 	}
-	ent = mdstat_by_component(devname);
+
+	mdstat = mdstat_read(0, 0);
+	if (!mdstat) {
+		pr_err("Cannot read /proc/mdstat file, aborting");
```

Comment:
Are you sure "\n" is not needed here? You marked it as resolved without comment.